src/content/docs/en/guides/deploy/space.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Deploy your Astro Site to Space
+description: How to build an Astro site in Space.
+type: deploy
+i18nReady: true
+---
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Deta Space](https://deta.space/developers) is a personal computer that lives in the cloud — a ‘personal cloud’. You can build and run apps for yourself in your own "Space". You can publish the apps you've built, and they'll run for people all around the world. 
+
+You can also install apps published by other devs. You'll get a personal copy of the app in your own cloud, running on your infrastructure. All the data that all these apps save to your Space is for you: to explore, to extend, and to export, if you like.
+
+This guide includes step-by-step instructions for building sites in Space. You can build any type of Astro site in Space: either a static site or a server-rendered site.
+
+## Prerequisite
+
+To push an Astro site to Space, make sure you first:
+
+- Create a [Space account](https://deta.space)
+- Install the [Space CLI](https://deta.space/docs/en/basics/cli) and log in
+
+No matter the type of Astro site you are building, create a Space project **inside** the directory of your Astro project. Run the CLI and follow the instructions on the screen.    
+
+```bash
+space new
+```
+
+:::note
+The Space CLI might prompt you with the `nodejs16` engine configuration. You may accept or disregard the recommendation, we will guide you through both options.
```

Comment:
Felt like this was a bit unclear. Simplified it by saying you should just accept the suggestions and follow the instructions.

```suggestion
The Space CLI will try to detect the right configuration for your app. Accept the suggested configuration and follow the instructions below on how to configure it for your Astro app.
```

Reply:
I like this way much better, thank you for the clarificaiton